test/e2e/apps/statefulset.go

```diff
@@ -850,8 +852,8 @@ var _ = SIGDescribe("StatefulSet", func() {
 			if err != nil {
 				framework.Failf("Failed to get scale subresource: %v", err)
 			}
-			framework.ExpectEqual(scale.Spec.Replicas, int32(1))
-			framework.ExpectEqual(scale.Status.Replicas, int32(1))
+			framework.ExpectEqual(scale.Spec.Replicas, int32(1), "")
+			framework.ExpectEqual(scale.Status.Replicas, int32(1), "")
```

Comment:
Unnecessary change, passing empty string is equal not to pass anything.

Reply:
@krmayankk advised we should add the check to guard against test flakiness. Which is our worst enemy in running these tests. https://github.com/kubernetes/kubernetes/pull/98126#discussion_r559879297
Should we keep it?